Docker Start failed on nightlys build 7288

Log File Location:
[https://storage.cloud.google.com/vic-ci-logs/functional_logs_vic_7288.zip?authuser=1&_ga=1.210014037.1122363360.1480435214](url)

Robot Log:

Running command 'docker -H sc-rdops-vm04-dhcp-109-114.eng.vmware.com:2376 --tls start 138707d442ee2b01cb8a534b8604c57e82aa93e8fe371b4951f8d5adc15cdd48 2>&1'.

${rc} = 1

${output} = Error response from daemon: Server error from portlayer: Get http://127.0.0.1:2377/containers/138707d442ee2b01cb8a534b8604c57e82aa93e8fe371b4951f8d5adc15cdd48: dial tcp 127.0.0.1:2377: getsockopt: con...

PortLayer Log:

```time=2016-12-02T08:20:23.197615894Z level=debug msg=[BEGIN] [github.com/vmware/vic/lib/portlayer/exec.Commit:35] 138707d442ee2b01cb8a534b8604c57e82aa93e8fe371b4951f8d5adc15cdd48 
time=2016-12-02T08:20:32.535414257Z level=debug msg=[ END ] [github.com/vmware/vic/lib/portlayer/exec.Commit:35] [9.337793326s] 138707d442ee2b01cb8a534b8604c57e82aa93e8fe371b4951f8d5adc15cdd48 
time=2016-12-02T08:20:32.547952320Z level=debug msg=[ END ] [github.com/vmware/vic/lib/apiservers/portlayer/restapi/handlers.(*ContainersHandlersImpl).CommitHandler:223] [9.352692274s] handle(d2814e2d12f849afd84664501ce76c34) 
time="2016-12-02T08:20:32Z" level=debug msg="Found 1 subscribers to events.ContainerEvent: Container 138707d442ee2b01cb8a534b8604c57e82aa93e8fe371b4951f8d5adc15cdd48 Created" 
time="2016-12-02T08:20:32Z" level=debug msg="Event manager calling back to netCtx(0xc42000f2c0)" 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0xcdfd17]

goroutine 18 [running]:
panic(0xec2240, 0xc420012050)
	/usr/local/go/src/runtime/panic.go:500 +0x1a1
github.com/vmware/vic/vendor/github.com/vmware/govmomi/event.(*eventTailer).newEvents(0xc42032490c, 0xc4205e8500, 0xa, 0x10, 0x1, 0x0, 0xe95460)
	/drone/src/github.com/vmware/vic/vendor/github.com/vmware/govmomi/event/processor.go:176 +0x57
github.com/vmware/vic/vendor/github.com/vmware/govmomi/event.(*eventProcessor).process(0xc4206a9da8, 0xc42026afe0, 0x15, 0xc420701260, 0x51, 0xc42092d080, 0x1, 0x4, 0x11, 0xc4208bea20)
	/drone/src/github.com/vmware/vic/vendor/github.com/vmware/govmomi/event/processor.go:144 +0x215
github.com/vmware/vic/vendor/github.com/vmware/govmomi/event.(*eventProcessor).run.func2(0xc42092d080, 0x1, 0x4, 0x0)
	/drone/src/github.com/vmware/vic/vendor/github.com/vmware/govmomi/event/processor.go:119 +0x9a
github.com/vmware/vic/vendor/github.com/vmware/govmomi/property.Wait.func1(0xc420112a60, 0x15, 0xc42000ed80, 0x51, 0xc42092d080, 0x1, 0x4, 0x0)
	/drone/src/github.com/vmware/vic/vendor/github.com/vmware/govmomi/property/wait.go:69 +0x43
github.com/vmware/vic/vendor/github.com/vmware/govmomi/property.waitLoop(0x1abe6e0, 0xc420012458, 0xc420202d20, 0xc4206a9898, 0x0, 0x0)
	/drone/src/github.com/vmware/vic/vendor/github.com/vmware/govmomi/property/wait.go:142 +0x1e5
github.com/vmware/vic/vendor/github.com/vmware/govmomi/property.Wait(0x1abe6e0, 0xc420012458, 0xc4206a9a50, 0xc42026afe0, 0x15, 0xc420701260, 0x51, 0xc4203249a0, 0x1, 0x1, ...)
	/drone/src/github.com/vmware/vic/vendor/github.com/vmware/govmomi/property/wait.go:70 +0x30a
github.com/vmware/vic/vendor/github.com/vmware/govmomi/event.(*eventProcessor).run(0xc4206a9da8, 0x1abe6e0, 0xc420012458, 0xc42077b601, 0x16, 0xc42077b637)
	/drone/src/github.com/vmware/vic/vendor/github.com/vmware/govmomi/event/processor.go:128 +0x90d
github.com/vmware/vic/vendor/github.com/vmware/govmomi/event.Manager.Events(0x0, 0x0, 0xc4200ea2c0, 0xc42066dd10, 0xc, 0xc42066dd40, 0xc, 0xc4203456e0, 0xc42075d6b8, 0xa, ...)
	/drone/src/github.com/vmware/vic/vendor/github.com/vmware/govmomi/event/manager.go:175 +0x1e0
github.com/vmware/vic/lib/portlayer/event/collector/vsphere.(*EventCollector).Start.func1(0x1abe6e0, 0xc420012458, 0x100000001, 0x1, 0xc42077b660, 0x1, 0x1, 0xc420345740, 0xc42066dd40, 0xc)
	/drone/src/github.com/vmware/vic/lib/portlayer/event/collector/vsphere/collector.go:139 +0xfd
created by github.com/vmware/vic/lib/portlayer/event/collector/vsphere.(*EventCollector).Start
	/drone/src/github.com/vmware/vic/lib/portlayer/event/collector/vsphere/collector.go:146 +0x316
time="2016-12-02T08:20:38Z" level=info msg="Launching portlayer server pprof server on 127.0.0.1:6063" 
time="2016-12-02T08:20:38Z" level=debug msg="secret key decoded from guestinfo.ovfEnv" 
time="2016-12-02T08:20:38Z" level=debug msg="Session Environment Info: " API Type=VirtualCenter API Version=6.5 Build=4602587 Name="VMware vCenter Server" OS Type=linux-x64 Product ID=vpx UUID=4315cc8a-78df-4b33-94f6-bc010ea77dde Vendor="VMware, Inc." Version=6.5.0```

--- a/pkg/vsphere/session/session.go
+++ b/pkg/vsphere/session/session.go
@@ -165,6 +165,8 @@ func (s *Session) Connect(ctx context.Context) (*Session, error) {
 	}
 
 	soapClient := soap.NewClient(soapURL, s.Insecure)
+	soapClient.Version = "6.0" // Pin to 6.0 until we need 6.5+ specific API
+
 	var login func(context.Context) error
 
 	login = func(ctx context.Context) error {

